core/domain/learner_progress_domain_test.py

```diff
@@ -0,0 +1,86 @@
+# coding: utf-8
+#
+# Copyright 2018 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Testing Domain objects for learner progress."""
+
+from core.domain import learner_progress_domain
+from core.tests import test_utils
+import feconf
+
+
+class LearnerProgressDomainUnitTests(test_utils.GenericTestBase):
+    """Test the Learner Progress domain."""
+
+    def setUp(self):
+        super(LearnerProgressDomainUnitTests, self).setUp()
+        self.exploration_keys_lst = []
```

Comment:
Let's drop all `_lst` since the variable name is already a plural. Therefore, change this to: `self.exploration_keys` (similarly elsewhere).